charts/jaeger/README.md

```diff
@@ -223,137 +225,139 @@ helm install jaeger jaegertracing/jaeger \
-| `serviceAccounts.spark.create` | Create service account | `true` |
-| `serviceAccounts.spark.name` | The name of the ServiceAccount to use. If not set and create is true, a name is generated using the fullname template | `` |
-| `spark.enabled` | Enables the dependencies job| `false` |
-| `spark.image` | Image for the dependencies job| `jaegertracing/spark-dependencies` |
-| `spark.pullPolicy` | Image pull policy of the deps image | `Always` |
-| `spark.schedule` | Schedule of the cron job | `"49 23 * * *"` |
-| `spark.successfulJobsHistoryLimit` | Cron job successfulJobsHistoryLimit | `5` |
-| `spark.failedJobsHistoryLimit` | Cron job failedJobsHistoryLimit | `5` |
-| `spark.tag` | Tag of the dependencies job image | `latest` |
-| `storage.cassandra.existingSecret` | Name of existing password secret object (for password authentication | `nil`
-| `storage.cassandra.host` | Provisioned cassandra host | `cassandra` |
-| `storage.cassandra.keyspace` | Schema name for cassandra | `jaeger_v1_test` |
-| `storage.cassandra.password` | Provisioned cassandra password  (ignored if storage.cassandra.existingSecret set) | `password` |
-| `storage.cassandra.port` | Provisioned cassandra port | `9042` |
-| `storage.cassandra.tls.enabled` | Provisioned cassandra TLS connection enabled | `false` |
-| `storage.cassandra.tls.secretName` | Provisioned cassandra TLS connection existing secret name (possible keys in secret: `ca-cert.pem`, `client-key.pem`, `client-cert.pem`, `cqlshrc`, `commonName`) | `` |
-| `storage.cassandra.usePassword` | Use password | `true` |
-| `storage.cassandra.user` | Provisioned cassandra username | `user` |
-| `storage.elasticsearch.existingSecret` | Name of existing password secret object (for password authentication | `nil` |
-| `storage.elasticsearch.host` | Provisioned elasticsearch host| `elasticsearch` |
-| `storage.elasticsearch.password` | Provisioned elasticsearch password  (ignored if storage.elasticsearch.existingSecret set | `changeme` |
-| `storage.elasticsearch.port` | Provisioned elasticsearch port| `9200` |
-| `storage.elasticsearch.scheme` | Provisioned elasticsearch scheme | `http` |
-| `storage.elasticsearch.usePassword` | Use password | `true` |
-| `storage.elasticsearch.user` | Provisioned elasticsearch user| `elastic` |
-| `storage.elasticsearch.nodesWanOnly` | Only access specified es host | `false` |
-| `storage.kafka.brokers` | Broker List for Kafka with port | `kafka:9092` |
-| `storage.kafka.topic` | Topic name for Kafka | `jaeger_v1_test` |
-| `storage.type` | Storage type (ES or Cassandra)| `cassandra` |
-| `tag` | Image tag/version | `1.16.0` |
+| Parameter                                                 | Description                                                                                                                                                      | Default                                 |
```

Comment:
Could we keep the spacing as it was before?
It is more readable in an IDE imho... BTW how did you space it all out?

Reply:
My IDE formats the markdown automatically.

Reply:
Oh I see, thanks.

Reply:
i have disabled the table autoformatting feature of my IDE and committed the changes

Reply:
Thanks! Hmmm maybe I should turn on mine. :)
 I will look into that but thanks for the info and reverting the change.